src/shared/Summary.ts

```diff
@@ -7,3 +7,7 @@ export const zSummary = z.object({
   });
 
 export type Summary = z.TypeOf<typeof zSummary>;
+
+export const zNameMap = z.record(z.string());
```

Comment:
Where do you use these variables? Also, again, use full name: zSummaryNameMap

Reply:
The SummaryNameMap type is used to define the nameMap variable in nameMapModal